### DISCUSSION TITLE
¡Únase a GitHub Educación!

### DISCUSSION THREAD

**Discussion by Marcela-Contreras** [community/community#125738] [GitHub Education]
### Hi everyone,

Hola buena tarde, mi Nombre es Marcela Contreras, desde hace 1 mes estoy intentando ingresar a la GitHub Educación, pero no se me a permitido, e subido mis comprobantes de estudio de la Universidad, carnet, certificación de estudio, certificación de notas, pero siempre me sale un error diferente, quisiera que por favor me pudieran ayudar, si es que en definitiva no puedo obtenerla y no intentar mas. Gracias

**Comment by Abhishek22K** [comment]
Aquí te dejo algunos pasos y recomendaciones que podrías seguir para intentar resolver este problema:

1. Revisa los Requisitos:
Asegúrate de cumplir con todos los requisitos de GitHub Education:

Debes ser estudiante de una institución académica acreditada.
Necesitas una dirección de correo electrónico de tu institución o documentación que pruebe tu estado de estudiante.
2. Documentación Correcta:
Verifica que los documentos que subiste sean claros y legibles.
Asegúrate de que tu nombre y la fecha sean visibles en todos los documentos.
Preferiblemente, sube un archivo PDF que contenga todos los documentos necesarios.
3. Usa el Correo Institucional:
Si tienes una dirección de correo electrónico de tu universidad, úsala para registrarte. Esto puede facilitar el proceso de verificación.
4. Error Común y Soluciones:
Si recibes errores específicos, toma nota de ellos. Algunos errores comunes incluyen problemas con la validación de documentos o con la cuenta de correo.
Intenta utilizar un navegador diferente o borra la caché y las cookies de tu navegador actual.
5. Contactar Soporte de GitHub:
Si continúas teniendo problemas, lo mejor es contactar directamente con el soporte de GitHub. Puedes hacerlo a través del siguiente enlace: [GitHub Support](https://support.github.com/contact).
Explica detalladamente el problema y adjunta capturas de pantalla de los errores que recibes.
6. Foros y Comunidad:
Consulta los foros de la comunidad de GitHub Education. Otros estudiantes pueden haber tenido problemas similares y podrías encontrar una solución allí.
Aquí está el enlace a la comunidad: GitHub Education Forum.
